src/RawRabbit/Common/NamingConventions.cs

```diff
@@ -64,15 +66,40 @@ private string GetSubscriberQueueSuffix(Type messageType)
 			return sb.ToString();
 		}
 
-		private string GetApplicationName()
+		private static string GetApplicationName()
 		{
-			return Directory
-				.GetCurrentDirectory()
-				.Split('\\')
-				.Select(d => d.ToLower())
-				.Last(d => !_disallowedDirectoryNames.Contains(d))
-				.Split('.')
-				.Last();
+			return GetApplicationNameFromCommandLine(Environment.CommandLine);
```

Comment:
You're right! Earlier I did some preparation in `GetApplicationName` before calling `GetApplicationNameFromCommandLine` but as for now this is unnecessary. I will remove it :)